### PR TITLE
Add `@parameter.outer` to php textobject

### PR DESCRIPTION
**Pull request by wilmanbarrios**

### CODE REVIEW

**Review by theHamsta — APPROVED** [review]
Thanks

File: queries/php/textobjects.scm [hunk 1]
```diff
@@ -25,3 +25,6 @@
 ;; parameters
 (formal_parameters
   (simple_parameter) @parameter.inner)
+
+(arguments
+  (argument) @parameter.outer)
```
Comment (stsewd) [inline queries/php/textobjects.scm:30]: I think we are marking these as `@parameter.inner`, outer would be including `,` separators

Comment (wilmanbarrios) [reply queries/php/textobjects.scm:30]: I'm not quite following you, can you explain a bit more.

Comment (stsewd) [reply queries/php/textobjects.scm:30]: You need to change this to `@parameter.innner`, outer is used when the capture includes the argument separator/whitespaces.

Comment (wilmanbarrios) [reply queries/php/textobjects.scm]: Thanks for being more clear :+1:

**Comment by wilmanbarrios** [comment]
Hey @theHamsta how do I tell the readme that `@parameter.outer` for PHP is supported?

**Comment by theHamsta** [comment]
A CI jobs will handle that. Just wait for it. It automatically pushes to master

**Comment by jessarcher** [comment]
Is it just me or did this PR only add `@parameter.inner` for function call arguments? It doesn't seem like `@parameter.outer` is currently supported?

**Comment by theHamsta** [comment]
This file contains only definitions for `@parameter.inner` https://github.com/nvim-treesitter/nvim-treesitter-textobjects/blob/d99036822287d817bb2e6a1c3a6e7586996d4da3/queries/php/textobjects.scm#L30-L25 Someone would need to add definitions for `@parameter.outer`

**Comment by greg0ire** [comment]
@jessarcher you're right, and moreover, there are 2 commits. In the second one, OP changes `outer` to `inner`… Hopefully, by looking at nearby files and learning about scm files in general, I will be able to figure out what needs to be done.

**Comment by greg0ire** [comment]
As a first step, I installed https://github.com/nvim-treesitter/playground.

Here is what the AST of the declaration of a method with 4 parameters looks like: 

```
    method_declaration [416, 4] - [502, 5]
      visibility_modifier [416, 4] - [416, 10]
      name: name [416, 20] - [416, 24]
      parameters: formal_parameters [416, 24] - [416, 80]
        simple_parameter [416, 25] - [416, 35]
          name: variable_name [416, 25] - [416, 35]
            name [416, 26] - [416, 35]
        simple_parameter [416, 37] - [416, 40]
          name: variable_name [416, 37] - [416, 40]
            name [416, 38] - [416, 40]
        simple_parameter [416, 42] - [416, 58]
          name: variable_name [416, 42] - [416, 51]
            name [416, 43] - [416, 51]
          default_value: null [416, 54] - [416, 58]
        simple_parameter [416, 60] - [416, 79]
          name: variable_name [416, 60] - [416, 72]
            name [416, 61] - [416, 72]
          default_value: null [416, 75] - [416, 79]
```

and here is what a function call with 2 arguments looks like:

```
                    arguments: arguments [418, 61] - [418, 79]
                      argument [418, 62] - [418, 72]
                        variable_name [418, 62] - [418, 72]
                          name [418, 63] - [418, 72]
                      argument [418, 74] - [418, 78]
                        string [418, 74] - [418, 78]
```

A method call looks similar, only the things before `arguments` change.

**Comment by greg0ire** [comment]
https://tree-sitter.github.io/tree-sitter/playground helped with figuring out what works, and copying code from the lua scm file seems to work just fine.

**Pull request merged**